GoogleForCreators/web-stories-wp
# Shopping: Editor: Honor Provider Dropdown

## Context



## Summary



## Relevant Technical Choices



## To-do



## User-facing changes

https://user-images.githubusercontent.com/237508/170541974-edcff205-7dfd-4a58-8fdf-7e976771b360.mov

## Testing Instructions






This PR can be tested by following these steps:

1. Activate woocemerce. 
2. Enable shopping experiment. 
3. Dashboard > settings > select woocemerce. 
4. Create a new story. 
5. Insert product in canvas. 
6. Save as draft.
7. Dashboard > settings > select none. 
8. Edit draft. 
9. Select product. See dropdown missing. 
10. Select product tab in library. 
11. See product list. 
12. Remove product. 
13. See product tab disappear. 


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #11266
